pvlib/inverter.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+"""
+This module contains functions for inverter modeling, primarily conversion of
+DC to AC power.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def sandia(v_dc, p_dc, inverter):
+    r'''
+    Convert DC power and voltage to AC power using Sandia's
+    Grid-Connected PV Inverter model.
+
+    Parameters
+    ----------
+    v_dc : numeric
+        DC voltage input to the inverter. [V]
+
+    p_dc : numeric
+        DC power input to the inverter. [W]
+
+    inverter : dict-like
+        Defines parameters for the inverter model in [1]_.  See Notes for
+        required model parameters. A copy of the parameter database from the
+        System Advisor Model (SAM) [2]_ is provided with pvlib and may be read
+        using :py:func:`pvlib.pvsystem.retrieve_sam.
+
+    Returns
+    -------
+    ac_power : numeric
+        AC power output. [W]
```

Comment:
I'd prefer `power_dc` but then I'd feel like we should change the argument names for `PVSystem.snlinverter`.

how strongly do you feel about this naming?

Reply:
> how strongly do you feel about this naming?

Not strongly at all, just saw a small inconsistency and pointed it out.  If the parameters do need to be renamed, might make the most sense to do it in a follow-on PR rather than this one.